src/java/com/unifina/signalpath/text/DecodeByteArrayToString.java

```diff
@@ -0,0 +1,63 @@
+package com.unifina.signalpath.text;
+
+import com.unifina.signalpath.*;
+import javax.xml.bind.DatatypeConverter;
+import java.util.*;
+
+public class DecodeByteArrayToString extends AbstractSignalPathModule {
+
+	private ByteArrayInput in = new ByteArrayInput(this, "in");
+	private StringOutput out = new StringOutput(this, "out");
+	private StringOutput error = new StringOutput(this, "error");
+	private DecodeParameter decodeType = new DecodeParameter(this, "decodeType", DecodeParameter.HEX);
+
+	@Override
+	public void init() {
+		addInput(in);
+		addInput(decodeType);
+		addOutput(out);
+		addOutput(error);
+		in.setCanConnect(true);
```

Comment:
Type `Input` does not have method `setCanConnect(boolean)`?

Reply:
it does